proto/RobotCommand.proto

```diff
@@ -5,7 +5,6 @@ package proto;
 
 message RobotCommand {
     uint32 id = 1;
-    bool active = 2;
```

Comment:
Are you 100% sure that this is unused in the rest of our code? Because I didn't see you remove this in the other PR's at all...

Reply:
I could not find it anywhere, it is not sent to the robot, and everything compiled fine after I removed it.

Reply:
Ok great